public/locales/es/translation.json

```diff
@@ -0,0 +1,163 @@
+
+{
+  "All": "Todos",
+  "box": {
+    "appupdate": {
+      "message": "Hay una nueva versión de Heroic disponible, ¿quieres actualizar ahora?",
+      "title": "Actualización disponible"
+    },
+    "choose": "Elegir",
+    "choose-egs-prefix": "Elija el Wine-Prefix donde está instalado EGS",
+    "default-install-path": "Elija la ruta de instalación predeterminada",
+    "error": {
+      "launch": "Error al iniciar el juego, ¡revisa los registros!"
+    },
+    "importpath": "",
+    "installpath": "",
+    "no": "No",
+    "quit": {
+      "message": "Hay operaciones pendientes, ¿estás seguro?",
+      "title": "Salir"
+    },
+    "runexe": {
+      "title": "Seleccione EXE para ejecutar"
+    },
+    "select": "Seleccionar",
+    "stopInstall": {
+      "keepInstalling": "",
```

Comment:
It think there are some missing translations on this file like this one. Can you check? 
Other than that looks fine 👍🏽

Reply:
@flavioislima I don't know what goes there, in all languages ​​it is empty including English :0
I have already changed the flags, the one for Spanish and Portuguese.

Reply:
Hmm, that's weird. I need to check then because maybe this is missing somewhere.

Reply:
You're right, those keys were duplicated for some reason. I'll fix soon.